### PR TITLE
Issue #PS-1631 feat: Add Telemetry Events in entire app

### PR DESCRIPTION
**Pull request by Aar-if**
Issue #PS-1631 feat: Add Telemetry Events in entire app

### CODE REVIEW

File: src/components/AddFacilitator.tsx [hunk 1]
```diff
@@ -281,6 +282,22 @@ const AddFacilitatorModal: React.FC<AddFacilitatorModalprops> = ({
                     userName: username,
                   });
 
+                  const telemetryInteract = {
+                    context: {
+                      env: 'facilitator-created-successfully',
```
Comment (itsvick) [inline src/components/AddFacilitator.tsx:287]: @Aar-if env cant be like this

Comment (Aar-if) [reply src/components/AddFacilitator.tsx:287]: What should I add here then ?

Comment (itsvick) [reply src/components/AddFacilitator.tsx:287]: teaching-center

File: src/components/AddLeanerModal.tsx [hunk 2]
```diff
@@ -223,6 +224,22 @@ const AddLearnerModal: React.FC<AddLearnerModalProps> = ({
               username: username,
             });
 
+            const telemetryInteract = {
+              context: {
+                env: 'learner-creation-success',
```
Comment (itsvick) [inline src/components/AddLeanerModal.tsx:229]: @Aar-if here as well

File: src/components/AddFacilitator.tsx [hunk 3]
```diff
@@ -281,6 +282,22 @@ const AddFacilitatorModal: React.FC<AddFacilitatorModalprops> = ({
                     userName: username,
                   });
 
+                  const telemetryInteract = {
+                    context: {
+                      env: 'teaching-center',
+                      cdata: [],
+                    },
+                    edata: {
+                      id: 'facilitator-created-successfully',
```
Comment (itsvick) [inline src/components/AddFacilitator.tsx:291]: @Aar-if change it to like facilitator-creation-success

File: src/components/AddLeanerModal.tsx [hunk 4]
```diff
@@ -223,6 +224,22 @@ const AddLearnerModal: React.FC<AddLearnerModalProps> = ({
...
+              context: {
+                env: 'teaching-center',
+                cdata: [],
+              },
+              edata: {
+                id: 'learner-creation-success',
+                type: 'CLICK',
+                subtype: '',
+                pageid: 'centers',
+                uid: localStorage.getItem('userId') ?? 'Anonymous',
```
Comment (itsvick) [inline src/components/AddLeanerModal.tsx:237]: @Aar-if no need to add this in every event right? if requires add this in telemetry.js Interact function and add there

File: src/components/AddLeanerModal.tsx [hunk 5]
```diff
@@ -223,6 +224,22 @@ const AddLearnerModal: React.FC<AddLearnerModalProps> = ({
...
             });
 
+            const telemetryInteract = {
+              context: {
+                env: 'teaching-center',
+                cdata: [],
+              },
+              edata: {
+                id: 'learner-creation-success',
+                type: 'CLICK',
```
Comment (itsvick) [inline src/components/AddLeanerModal.tsx:234]: @Aar-if create enum for type here

File: src/components/center/centerList.tsx [hunk 6]
```diff
@@ -0,0 +1,123 @@
+import { CenterType } from "@/utils/app.constant";
+import { Box, Grid } from "@mui/material";
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import SmartDisplayOutlinedIcon from '@mui/icons-material/SmartDisplayOutlined';
+import building from '../../assets/images/apartment.png';
+import Image from "next/image";
+
+interface Center {
```
Comment (itsvick) [inline src/components/center/centerList.tsx:8]: @Aar-if looks like this is not related to telemetry

File: src/utils/telemetry.js [hunk 7]
```diff
@@ -36,9 +36,11 @@ const telemetryConfig = {
   host: hostURL,
   endpoint: '/telemetry/v1/telemetry',
   tags: [],
-  enableValidation: true
+  enableValidation: false
```
Comment (itsvick) [inline src/utils/telemetry.js:39]: @Aar-if enable it and resolve all the issues listed in the console

File: src/components/AddFacilitator.tsx [hunk 8]
```diff
@@ -281,6 +282,22 @@ const AddFacilitatorModal: React.FC<AddFacilitatorModalprops> = ({
                     userName: username,
                   });
 
+                  const telemetryInteract = {
+                    context: {
+                      env: 'teaching-center',
+                      cdata: [],
+                    },
+                    edata: {
+                      id: 'facilitator-created-success',
```
Comment (itsvick) [inline src/components/AddFacilitator.tsx:291]: @Aar-if it should be `facilitator-creation-success`

File: src/components/AddFacilitator.tsx [hunk 9]
```diff
@@ -281,6 +282,22 @@ const AddFacilitatorModal: React.FC<AddFacilitatorModalprops> = ({
...
+                      type: Telemetry.CLICK,
+                      subtype: '',
+                      pageid: 'centers',
+                      uid: localStorage.getItem('userId') ?? 'Anonymous',
```
Comment (itsvick) [inline src/components/AddFacilitator.tsx:295]: @Aar-if move this uid and username to the common function only
